src/host/telemetry/telemetry_retriever_border_agent.hpp

```diff
@@ -26,6 +26,11 @@
  *    POSSIBILITY OF SUCH DAMAGE.
  */
 
+#ifndef OTBR_TELEMETRY_RETRIEVER_BORDER_AGENT_HPP_
+#define OTBR_TELEMETRY_RETRIEVER_BORDER_AGENT_HPP_
```

Comment:
Correct me if I'm wrong.
```suggestion
#ifndef OTBR_AGENT_TELEMETRY_RETRIEVER_BORDER_AGENT_HPP_
#define OTBR_AGENT_TELEMETRY_RETRIEVER_BORDER_AGENT_HPP_
```